src/AyonUsdResolver/cache/resolverContextCache.h

```diff
@@ -1,17 +1,35 @@
 #ifndef AR_AYONUSDRESOLVER_RESOLVER_CONTEXT_CACHE_H
 #define AR_AYONUSDRESOLVER_RESOLVER_CONTEXT_CACHE_H
 
+#include <filesystem>
+#include <optional>
 #include <shared_mutex>
 #include <string>
-#include <unordered_set>
+#include <unordered_map>
 
 #include "AyonCppApi.h"
-#include "assetIdentDef.h"
+#include "../cache/assetIdentDef.h"
+#include <nlohmann/json.hpp>
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
 enum cacheName { AYONCACHE, COMMONCACHE };
 
+class pinningFileHanlder {
+    public:
+        pinningFileHanlder(const std::string &pinningFilePath,
+                           const std::unordered_map<std::string, std::string> &rootReplaceData);
+        ~pinningFileHanlder() = default;
```

Comment:
```suggestion
class pinningFileHandler {
    public:
        pinningFileHandler(const std::string &pinningFilePath,
                           const std::unordered_map<std::string, std::string> &rootReplaceData);
        ~pinningFileHandler() = default;
```
Typos. Of course also check other usages of these.

Reply:
Fixed via a commit.